packages/coachmark/test/coachmark.test.ts

```diff
@@ -10,30 +10,345 @@ OF ANY KIND, either express or implied. See the License for the specific languag
 governing permissions and limitations under the License.
 */
 
-import { elementUpdated, expect, fixture, html } from '@open-wc/testing';
+import {
+    elementUpdated,
+    expect,
+    fixture,
+    html,
+    nextFrame,
+} from '@open-wc/testing';
 
-import '@spectrum-web-components/coachmark/sp-coachmark.js';
-import { Coachmark } from '@spectrum-web-components/coachmark';
+import { Coachmark, CoachmarkItem } from '@spectrum-web-components/coachmark';
 import { testForLitDevWarnings } from '../../../test/testing-helpers.js';
+import '@spectrum-web-components/coachmark/sp-coachmark.js';
+import '@spectrum-web-components/coachmark/sp-coach-indicator.js';
+import '@spectrum-web-components/overlay/sp-overlay.js';
+import { spy } from 'sinon';
+import { Button } from '@spectrum-web-components/button';
 
+const defaultItem: CoachmarkItem = {
```

Comment:
There's a bunch of new DOM and data in this file, is any of it similar enough to that found in the stories for this component to be shared between the two? The less copies and slight versions of code in the test suite the easier it will be to refactor/maintain this over time.

Reply:
Refactored the tests and also the stories.